Websites/bugs.webkit.org/template/en/custom/bug/create/create.html.tmpl

```diff
@@ -154,8 +193,12 @@ TUI_hide_default('attachment_text_field');
     </div>
     [% END %]
     
-    <p><em>Note: If you are reporting an intermittent issue that WebKit developers may not be able to easily reproduce, please also file a <a href="https://feedbackassistant.apple.com">Feedback Assistant</a> report to <a href="https://developer.apple.com/bug-reporting/profiles-and-logs/?name=sysdiagnose">include a sysdiagnose</a>.</em></p>
-
+    <h4>Please note:</h4>
+    <ul>
+      <li>Add testcases as an attachment or quote problematic code in the bug report even when providing a link for convenience to guarantee WebKit engineers can review buggy behavior.</li>
```

Comment:
```suggestion
    <h4>Please note:</h4>
    <ul>
      <li>Add testcases as an attachment or quote problematic code in the bug report even when providing a link to guarantee WebKit engineers can review buggy behavior, as linked pages can change or be unavailable later.</li>
```

Suggest explaining why so that there's some incentive to do it. :) Other thoughts:

* Can we come up with a better title than Please Note? (Should there even be a heading?)
* Do we want to start with an imperative here or something like "It's best to" or "Please" or "It helps if" or something?